site/_includes/macros/landing-section.njk

```diff
@@ -24,18 +24,18 @@
               <div class="item">
                 <div class="item__inner height-full display-flex direction-column">
                   <div class="item__title">
-                    <a href="{{ post.url }}">{{ post.title or post.data.title }}</a>
+                    <a href="{{ post.url }}">{{ post.title or post.data.title | md | safe }} </a>
```

Comment:
Once https://github.com/GoogleChrome/developer.chrome.com/pull/4431 lands, can we add examples of this component,  including markdown, to document & test that markdown is allowed and working?

Reply:
Yes, we can add the example of using markdown with this component into the 'add a landing page' page